fixposition_driver_ros2:
  ros__parameters:
    fp_output:
      formats: [
                "ODOMETRY", "LLH", "ODOMENU", "ODOMSH", "ODOMSTATUS", "RAWIMU", "CORRIMU", 
                "IMUBIAS", "GNSSANT", "GNSSCORR", "EOE", "TEXT", "TF", "TP", # FP_A
                "GPGGA", "GPGLL", "GNGSA", "GPGST", "GPHDT", "GPRMC", "GPVTG", "GPZDA", "GXGSV" # NMEA
               ]
      type: "serial"
      port: "/dev/ttyUSB0"
      baudrate: 115200
      rate: 200
      reconnect_delay: 5.0 # wait time in [s] until retry connection
      cov_warning: false
      nav2_mode: false

    customer_input:
      speed_topic: "/fixposition/speed"
      rtcm_topic: "/rtcm"
